projects/fishtornado/Flocking.h

```diff
@@ -64,8 +65,9 @@ class Flocking
         bool renderedWithAsyncCompute = false;
     };
 
-    uint32_t mResX;
-    uint32_t mResY;
+    const FishTornadoSettings* pSettings;
```

Comment:
Is there any particular reason to hold on to the settings instead of fishThreadsX and fishThreadsY?

Reply:
It's actually not the threads that's problematic - it's the enable/disable head tracking specifically. Because that's a live-updatable setting in the main FishTornado.cpp, it can't be stored off. I could do a pointer specifically for that setting, but I also figured it would be easy enough to keep a pointer to the entire settings structure. Thoughts?

Reply:
Ah, yes, I thought threads that are updatable but its the head tracking, I was looking at the non-xr code.
I guess we could have had something like a method on flocking to set/update parameters, but I don't know if that's better. This is fine then, perhaps a comment somewhere to document this might help?

Reply:
I think it's probably better to separately track head info, given that the non-XR version doesn't need it. Let me clean this up a bit.